Source/relay/TourGuide/Items of the Month/2023/Cursed Monkey Paw.ash

```diff
@@ -0,0 +1,260 @@
+        locationAvailable($location[The Haunted Library])
+    ),
+    new Wishable(
+        $item[none],
+        $effect[Dirty Pear],
+        HTMLGenerateSpanFont("double sleaze damage", "purple"),
+        get_property_int("zeppelinProtestors") < 80,
+        true
+    ),
+    new Wishable(
+        $item[glark cable],
+        $effect[none],
+        "",
+        __quest_state["Level 11 Ron"].mafia_internal_step < 5,
+        locationAvailable($location[The Red Zeppelin])
+    ),
+    new Wishable(
+        $item[short writ of habeas corpus],
+        $effect[none],
+        "",
+        $item[spectre scepter].available_amount() < 1,
+        locationAvailable($location[The Hidden Park])
+    ),
+    new Wishable(
+        $item[lion oil],
+        $effect[none],
+        "",
+        $item[mega gem].available_amount() < 1 &&
+            $item[lion oil].available_amount() < 1,
+        locationAvailable($location[Whitey's Grove])
+    ),
+    new Wishable(
+        $item[bird rib],
+        $effect[none],
+        "",
+        $item[mega gem].available_amount() < 1 &&
+            $item[bird rib].available_amount() < 1,
+        locationAvailable($location[Whitey's Grove])
+    ),
+    new Wishable(
+        $item[drum machine],
+        $effect[none],
+        "",
+        get_property_int("desertExploration") < 100 &&
+            $item[drum machine].available_amount() < 1,
+        locationAvailable($location[The Oasis])
+    ),
+    new Wishable(
+        $item[green smoke bomb],
+        $effect[none],
+        "",
+        __quest_state["Level 12"].state_string["Side seemingly fighting for"] != "hippy",
+        !__quest_state["Level 12"].finished &&
+            locationAvailable($location[The Battlefield (Frat Uniform)])
+    ),
+    new Wishable(
+        $item[star chart],
+        $effect[none],
+        "",
+        !__quest_state["Level 13"].state_boolean["Richard's star key used"] &&
+            $item[Richard's star key].available_amount() < 1 &&
+            $item[star chart].available_amount() < 1,
+        locationAvailable($location[The Hole In The Sky])
+    ),
+    new Wishable(
+        $item[none],
+        $effect[Frosty],
+        "item/meat for 8-bit",
+        !__quest_state["Level 13"].state_boolean["digital key used"] &&
+            $item[digital key].available_amount() < 1 &&
+            get_property("8BitScore") < 10000,
+        true
+    ),
+    new Wishable(
+        $item[lowercase N],
+        $effect[none],
+        "",
+        !__quest_state["Level 13"].state_boolean["king waiting to be freed"] &&
+            $item[wand of nagamar].available_amount() < 1 &&
+            $item[lowercase N].available_amount() < 1,
+        locationAvailable($location[The Valley of Rof L'm Fao])
+    )
+};
+
+Wishable [int] aftercoreWishables = {
+    new Wishable(
+        $item[bag of foreign bribes],
+        $effect[none],
+        "",
+        locationAvailable($location[The Ice Hotel]),
+        true
+    )
+};
```

Comment:
We could flesh this out more, but I assume a lot of people are either going to let garbo handle it or have highly custom setups.